docs/core/deploying/native-aot/index.md

```diff
@@ -35,7 +35,7 @@ On Linux, install clang and developer packages for libraries that .NET runtime d
 - Alpine (3.15+)
 
     ```sh
-    sudo apk add clang gcc lld musl-dev build-base zlib-dev
+    sudo apk add build-base zlib-dev
```

Comment:
Line 27 notes `install clang and developer packages`, but `build-base` on Alpine (and `build-essential` on Debian/Ubuntu) is a meta package which include gcc toolchain but not llvm-toolchain. Should we update the note to `install compiler toolchain and developer packages`.

Alternatively:

```suggestion
    sudo apk add clang build-base zlib-dev
```